docs/authentication/social-connections/coinbase.mdx

```diff
@@ -1,38 +1,78 @@
 
-## Configuring Coinbase social connection
+<Steps>
+  ### Enable Coinbase as a social connection in Clerk
 
-First, you need to create a new OAuth2 Coinbase app. You can find this option in your **Profile > Settings > API > New OAuth2 Application**.
+  1. In the Clerk Dashboard, navigate to the [**SSO Connections**](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections) page.
+  1. Select **Add connection** and select **For all users**.
+  1. In the **Choose provider** dropdown, select **Coinbase**.
+  1. Ensure that both **Enable for sign-up and sign-in** and **Use custom credentials** are toggled on.
+  1. Save the **Redirect URI** somewhere secure. Keep the modal and page open.
 
-You need to set all the required fields and the corresponding **Permitted Redirect URIs**. Under **Advanced Settings**, make sure to also select **Reuse Existing Tokens** for **Previously-Authorized Users** so that your users won't get notified everytime they sign in with their Coinbase account. Then, click **Create application**.
+  ### Create a Coinbase app
 
-![Creating application](/docs/images/authentication-providers/coinbase/creating-application.jpg)
+  > [!NOTE]
+  > Coinbase automatically creates a default project for you named `Project 1`. Select the icon next to the project name to rename.
 
-Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections). In the top navigation, select **Configure**. Then in the sidebar, select **SSO Connections**. Select the **Add connection** button, and select **For all users**. In the **Choose provider** dropdown, select **Coinbase**. Toggle on **Use custom credentials** and copy **Redirect URI**. Make sure the value matches the **Redirect URIs**, as shown below.
+  1. In your [Coinbase Developer Platform project dashboard](https://portal.cdp.coinbase.com/projects), navigate to the **API Keys** tab.
+  1. Select **OAuth**, then select **Create client**.
+  1. Complete the required fields. In **Redirect URIs**, paste the **Redirect URI** you saved from the Clerk Dashboard.
+  1. Select **Create client**. The page will refresh and you should see the **Client ID** and **Client Secret** in the **OAuth** section. Save both values somewhere secure. Keep this page open.
 
-![Copying the Client ID and Client Secret from the Box dashboard](/docs/images/authentication-providers/coinbase/copying-values-from-box-dashboard.jpg)
+  ### Set the Client ID and Client Secret in your Clerk Dashboard
 
-Copy the **Client ID** and **Client secret** as shown in the above image. Go back to the Clerk Dashboard and paste them into the respective fields.
+  1. Navigate back to your Clerk Dashboard where the modal should still be open. Paste these values into the respective fields.
```

Comment:
we can't say "these" because we don't specify which values, in this section. 

```suggestion
  1. Navigate back to your Clerk Dashboard where the modal should still be open. Paste the **Client ID** and **Client Secret** values into the respective fields.
```